### ISSUE TITLE
Readme thanks section incomplete

### ISSUE THREAD

**Issue by jasonkneen** [CanalTP/grunt-alloy-theme-switcher#1]
Looks like it's missing a mention of @jasonkneen who wrote TiCh, which *seems* to have inspired this project (developer forked TiCh in January)

(would have welcomed some pull requests / issues etc in the TiCh repo to add this support btw)

**Comment by yuchi** [comment]
Both *TiCh* and *TiTh* are licensed under the Apache License 2.0, so while derivative work is permitted, also you need to explicitly state the attribution where relevant, and what is a derivative is described as follows:

> "Derivative Works" shall mean any work, whether in Source or Object form, that is based on (or derived from) the Work and for which the editorial revisions, annotations, elaborations, or other modifications represent, as a whole, an original work of authorship. For the purposes of this License, Derivative Works shall not include works that remain separable from, or merely link (or bind by name) to the interfaces of, the Work and Derivative Works thereof.


*tl;dr*: if this project it’s in some form a derivative of *TiCh* or *TiTh* you should make it clear. Which would be also quite sort of nice to Jason too.

**Comment by jasonkneen** [comment]
+1

**Comment by vinzentt** [comment]
Hi,

no pb to mention both tich and tith in the thanks section. Done.

Concerning the PR I didn't do. I first tried to require tich in my grunt task but it was not usable as so because it is not written as a common js module. So I started to rewrite it but it was like rewriting the whole file, and then include it via a grunt task. It looked like overkill to me. Plus a PR consisting in changing almost every lines in a project had not so many chances to be accepted... I'm trying to do as many PR as possible as you can see here https://github.com/FokkeZB/ti-i18n/pull/13

Anyway this oversight was not intentional and I agree on your point, but since it's a small part (20 lines of "inspired" code at the most) of what my plugin does (and will do), I was more fussy about thanking tiapp.xml and ti-i18n modules on which this plugin relies.

Sorry for this oversight.
